packages/render-fidelity-tools/src/components/renderers/filament-viewer.ts

```diff
@@ -14,13 +14,18 @@
  */
 
 import {resolveDpr} from '@google/model-viewer/lib/utilities.js';
+import {Camera, Engine, IndirectLight, Renderer, Scene, SwapChain, View} from 'filament';
+import {Entity, EntityManager, gltfio$FilamentAsset, Skybox} from 'filament';
+import {Camera$Fov, LightManager, LightManager$Type} from 'filament';
+import {Aabb, init} from 'filament';
+import {fetch as filamentFetch} from 'filament';
```

Comment:
Let's merge all of these into a single import. I know it's kind of a mess with all of these individual imports, but it's more consistent with what we do for e.g., `three`.

Reply:
Fixed.